presto-main/src/main/java/com/facebook/presto/execution/buffer/SerializedPage.java

```diff
@@ -85,13 +85,31 @@ public Slice getSlice()
     }
 
     @Override
-    public String toString()
+    public boolean equals(Object o)
```

Comment:
`equals` and `hashCode` didn't exist in `SerializedPage` previously , right?

Reply:
> `equals` and `hashCode` didn't exist in `SerializedPage` previously , right?

Right